cmds/dump.c

```diff
@@ -1058,8 +1058,8 @@ static void adjust_fg_time(struct uftrace_task_graph *tg, void *arg)
 	 */
 	accounted_time = (curr_time / sample_time) * sample_time;
 
-	node->parent->child_time -= curr_time;
-	node->parent->child_time += accounted_time;
+	node->parent->self_time.sum -= curr_time;
+	node->parent->self_time.sum += accounted_time;
```

Comment:
This looks suspicious... why did you replace child time with self time?

Reply:
I changed the name to match the term with `uftrace_report_node`.
```
struct uftrace_report_node {
    char *name;
    struct report_time_stat total;
    struct report_time_stat self;
    struct uftrace_dbg_loc *loc;
    uint64_t call;
    struct rb_node name_link;
    struct rb_node sort_link;

    /* used by diff */
    struct uftrace_report_node *pair;
};
```
https://github.com/namhyung/uftrace/blob/master/utils/report.h#L25-L36

Reply:
Okay, I'll change it to that.